shellbase-core/src/main/scala/com/sumologic/shellbase/ShellBase.scala

```diff
@@ -318,21 +323,38 @@ abstract class ShellBase(val name: String) {
 
   val subCommandExtractor = ShellBase.SubCommandExtractor
 
-  rootSet.commands += new ClearCommand
+  rootSet.commands += new ClearCommand {
```

Comment:
At some point in the future, I think we should find a way to allow these built in ones to be added/removed and not just suppressed from help.